website/docs/guides/core-to-cloud-1.md

```diff
@@ -15,44 +16,50 @@ recently_updated: true
 
 ## Introduction
 
-Moving from dbt Core to dbt Cloud streamlines analytics engineering workflows by allowing teams to develop, test, deploy, and explore data products using a single, fully managed software service.
+Moving from dbt Core to dbt Cloud streamlines analytics engineering workflows by allowing teams to develop, test, deploy, and explore data products using a single, fully managed platform.
+
+Explore our 3-part-guide series on moving from dbt Core to dbt Cloud. The series is ideal for users aiming for streamlined workflows and enhanced analytics:
```

Comment:
Is it worth numbering these guides for anyone who fills all of those roles? Should they start with "Get Started" or "What you need to know" first or do we consider that order irrelevant? 

```suggestion
Explore our 3-part-guide series on moving from dbt Core to dbt Cloud. This series is ideal for users aiming for streamlined workflows and enhanced analytics:
```

Reply:
yea i initially ordered them but then removed it bc i think the order will depend on the user role. going to cc @jtcohen6 to see his thoughts on numbering them or leaving them as bullets